.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,58 @@
+name: Bug report
+description: Report to help us reproduce the bug
+labels: ['Needs Triage']
+title: "[Bug]: "
+
+body:
+- type: markdown 
+  attributes:
+    value: >
+        ## Thanks for taking the time to fill out this report
+- type: markdown 
+  attributes:
+    value: >
+       ### Before submitting a bug, please make sure it has not been reported in the 
+       [list of issues](https://github.com/narwhals-dev/narwhals/issues).
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >
+       A clear description of the bug.
+  validations:
+       required: true
+- type: textarea
+  attributes:
+    label: Steps or code to reproduce the bug
+    description: |
+       Please add a minimal code example to reproduce the bug.
+  validations:
+       required: true 
+- type: textarea
+  attributes:
+    label: Expected Results
```

```suggestion
    label: Expected results
```
Let's keep the formatting of the heading consistent with the rest of the template.

---

Same. :-)